src/server/models/listItem/summary.ts

```diff
@@ -123,6 +167,9 @@ export async function findIndexListItems(options: ListIndexOptions): Promise<
         },
       ],
     },
+    include: {
+      history: true,
```

Comment:
it's possible we end up loading and processing a lot of data when we just need the most recent publish event. 
```
 history: {
   take: 1, // double check this, but it should work iirc, limit query to 1
   where: {
     type: "PUBLISHED",
   },
   orderBy: {
     time: "desc",
   },
},
```